### PR TITLE
Refactor InfluxDB scaler config

### PR DESCRIPTION
**Pull request by dovics**
Refactor the scaler to match the new metadata guidelines.

### Checklist

- [x] I have verified that my change is according to the [deprecations & breaking changes policy](https://github.com/kedacore/governance/blob/main/DEPRECATIONS.md)
- [x] Tests have been added
- [x] Changelog has been updated and is aligned with our [changelog requirements](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#Changelog)
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

Fixes #

Related: https://github.com/kedacore/keda/pull/5797

### CODE REVIEW

File: pkg/scalers/influxdb_scaler.go [hunk 1]
```diff
@@ -23,14 +22,15 @@ type influxDBScaler struct {
...
-	serverURL                string
-	unsafeSsl                bool
-	thresholdValue           float64
-	activationThresholdValue float64
-	triggerIndex             int
+	AuthToken                string  `keda:"name=authToken, order=triggerMetadata;resolvedEnv;authParams"`
+	OrganizationName         string  `keda:"name=organizationName, order=triggerMetadata;resolvedEnv;authParams"`
+	Query                    string  `keda:"name=query, order=triggerMetadata"`
+	ServerURL                string  `keda:"name=serverURL, order=triggerMetadata;authParams"`
+	UnsafeSsl                bool    `keda:"name=UnsafeSsl, order=triggerMetadata, optional"`
```
Comment (JorTurFer) [inline pkg/scalers/influxdb_scaler.go:29]:
```suggestion
	UnsafeSsl                bool    `keda:"name=unsafeSsl, order=triggerMetadata, optional"`
```

**Review by JorTurFer — APPROVED** [review]
Looking good, just a single type inline. PTAL @wozniakjan

File: pkg/scalers/influxdb_scaler.go (hunk 1 above)
Comment (dovics) [reply pkg/scalers/influxdb_scaler.go]: updated

**Review by wozniakjan — APPROVED** [review]
lgtm, thank you!

**Pull request merged**